### PR TITLE
Predefined filtering

### PR DESCRIPTION
**Pull request by JakobNH**
## Background

We wanted a kind of "Fill mode" where the user gets fewer columns to worry about when answering, to make it nicer to answer.

## Solution

Implement the solution using the states we had + some helper functions

Screenshot 2024-08-06 at 08 54 52

### CODE REVIEW

File: frontend/beCompliant/src/components/table/DataTable.tsx [hunk 1]
```diff
@@ -71,7 +99,42 @@ export function DataTable<TData>({
...
           </Flex>
         )}
-        {showSearch && <DataTableSearch table={table} />}
+        <Flex alignItems={'center'} gap={3}>
+          <Flex
+            alignItems={'center'}
+            flexDir={'row'}
+            backgroundColor={theme.colors.blue[50]}
+            padding={4}
+            borderRadius="5"
```
Comment (KrisAesoey) [inline frontend/beCompliant/src/components/table/DataTable.tsx:108]: {5} eller "5px"?

**Review by KrisAesoey — APPROVED** [review]

**Comment by SondreKanstad** [comment]
Jeg fikk en merkelig bug når jeg klikka på masse greier 😅 

https://github.com/user-attachments/assets/6b8550dc-631d-4d79-ac1c-6e5747805831

**Comment by JakobNH** [comment]
> Jeg fikk en merkelig bug når jeg klikka på masse greier 😅
> 
>  Screen.Recording.2024-08-06.at.09.25.11.mov

Oi! Hmm 🤔 ser på det

**Review by SondreKanstad — APPROVED** [review]
LGTM

**Pull request merged**